README.md

```diff
@@ -42,29 +57,23 @@ You can find the Clay documentation on the [site](https://clayui.com/docs).
 
 You can improve by sending pull requests to this repository.
 
-## Installation
-
-Clay is available as the `clay` package in npm for components and for css like `clay-css`. It is also available on a [CDN](https://cdn.jsdelivr.net/npm/clay/lib/js/clay.js) for components and for css, [check](https://cdn.jsdelivr.net/npm/clay/lib/css/atlas.css). But if you just want the separate components, it's also available in the npm as [clay-](https://www.npmjs.com/search?q=clay-).
-
-See the [Getting Started](https://clayui.com/docs/get-started/index.html) guide for more information.
-
 ## Migration Guides
 
 Do you already have the Clay v2 components implemented in your project? These handy guides will help you perform the migration from [Clay v2](https://v2.clayui.com) to [v3](https://clayui.com).
 
 -   [Migrate the Clay components from v2 to v3](https://clayui.com/docs/get-started/migrate-the-clay-components-from-v2-to-v3.html)
 
-## Contributing
+## Installation
 
-Feel free to open up problems or send pull requests. We will always be looking at these problems and we will be responding whenever possible.
+Clay is available as the `clay` package in npm for components and for css like `clay-css`. It is also available on a [CDN](https://cdn.jsdelivr.net/npm/clay/lib/js/clay.js) for components and for css, [check](https://cdn.jsdelivr.net/npm/clay/lib/css/atlas.css). But if you just want the separate components, it's also available in the npm as [clay-](https://www.npmjs.com/search?q=clay-).
```

Comment:
The `clay` package in npm, is an older version of Clay as far as I can remember. 
Maybe we don't need to mention this (I wouldn't). Also, and just so you know and since this isn't easy to understand as someone "new" on the team: 

- In clay `2.x`, we used Metal.js and Soy and published the components with the following naming convention: `clay-COMPONENT_NAME`, for example: `clay-button`
- In clay `3.x`, we now use React (and TypeScript) and packages are published with the following naming convention: `@clayui/COMPONENT_NAME`, for example: `@clayui/button`

Reply:
If this point (Installation) explains `clay` npm package and it is an old version that someone new doesn't understand, I have two choices:

- Delete **Installation** point because it explains several concepts that are not easy to understand.
- In **Migration Guides** point I can add a small section with some information about clay `2.x` and clay `3.x`.

What do you think @julien?

Reply:
I would go for 

> Delete Installation point because it explains several concepts that are not easy to understand.

Simply because no one is going to understand :rofl: 

Clay is available as "various" packages, there's actually one package per component, although that might change in the near future (see #4191 when you have time) 

Since this is the 3.x branch, I'd avoid mentioning anything about 2.x anyway, since it's just there for maintenance purposes or security fixes.

On top of that, anyone familiar with npm/React should be able to see very easily what the name (in the npm registry) of a component is by looking at the corresponding package.json file in the component's directory.